cmd/controller/controller.go

```diff
@@ -215,15 +216,8 @@ func (c *command) start(ctx context.Context) error {
 	logrus.Infof("using storage backend %s", nodeConfig.Spec.Storage.Type)
 	nodeComponents.Add(ctx, storageBackend)
 
-	controllerLeaseCounter := &controller.K0sControllersLeaseCounter{
-		InvocationID:      c.K0sVars.InvocationID,
-		ClusterConfig:     nodeConfig,
-		KubeClientFactory: adminClientFactory,
-	}
-
-	if !c.SingleNode {
-		nodeComponents.Add(ctx, controllerLeaseCounter)
-	}
+	// Assume a single active controller during startup
+	numActiveControllers := value.NewLatest[uint](1)
```

Comment:
Doesn't this mean that on the second controller, konnectivity would start once with `1` and then get restarted with `2` right away?